Sources/Turf/Geometries/Point.swift

```diff
@@ -14,3 +14,27 @@ public struct Point: Equatable {
         self.coordinates = coordinates
     }
 }
+
+extension Point: Codable {
+    enum CodingKeys: String, CodingKey {
+        case kind = "type"
+        case coordinates
+    }
+    
+    enum Kind: String, Codable {
+        case Point
+    }
+    
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        _ = try container.decode(Kind.self, forKey: .kind)
+        let coordinates = try container.decode(LocationCoordinate2DCodable.self, forKey: .coordinates).decodedCoordinates
+        self = .init(coordinates)
+    }
+    
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(Kind.Point, forKey: .kind)
+        try container.encode(coordinates.codableCoordinates, forKey: .coordinates)
+    }
+}
```

Comment:
Each of the geometric types has a very similar Codable implementation. In the interest of time, I’m leaving this redundancy in place, but it would be nice to figure out a more generic implementation at some point. The sticking point is that everything is ultimately based on `CLLocationCoordinate2D`, which we can’t conform to Codable because it would potentially collide with other libraries that use that standard type in a non-GeoJSON context: #84.